QuizAppNew/View/QAview.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  QAview.swift
+//  QuizAppNew
+//
+//  Created by Jannatun Nahar Papia on 20/8/22.
+//
+
+import SwiftUI
+
+//MARK: Question Answer view
+struct QAview: View {
+    //outgoing
+    @Binding var correct: Int
+    @Binding var wrong: Int
+    @Binding var answered: Int
+    var set : String
+    @State var data = Qustion()
+    
+//    @State var answered: Int = 0
+    var body: some View {
+        
+        ZStack {
+            
```

Comment:
**data.question.isEmpty** why logic in side if remove it. no longer need a progress view.  you are only responsible for making UI design.

Reply:
logic is remove .. Only making UI